private4/index.php

```diff
@@ -74,15 +74,9 @@ function updateForHash() {
 
   <a href="https://genealogiskforum.dk/kontakt/"style="font-family: arial; font-weight: normal; font-size: 12px; color: lightgrey; margin-left:10px; class="w3-bar-item w3-button">Kontakt </a> <br>
 
-<br>	
+<br>
 
-<script>
-function myNavi() {
-    document.getElementById("navigatorDiv").innerHTML;
```

Comment:
This line does nothing, so this function does nothing, so it's not worth even calling, so I got rid of it.